Project/src/MakeCall/RemoteParticipantCard.js

```diff
@@ -186,9 +204,9 @@ export default class RemoteParticipantCard extends React.Component {
                         <div className="inline-block">
                         {
                             this.state.state === "InLobby" ?
-                                <div className="text-right lobby-action">
-                                    <a href="#" onClick={e => this.admitParticipant(e)} className="float-right ml-3"> Admit Participant</a>
-                                    <a href="#" onClick={e => this.rejectParticipant(e)} className="float-right ml-3"> Reject Participant</a>
+                                <div className="text-right lobby-action" id="lobbyAction" hidden={this.state.canManageLobby === false}>
```

Comment:
Why is canManageLobby on a remote participant?

Reply:
User could admit/reject remote participant who has inlobby state, the admit/reject button are in the remoteparticipant card, so user could simply click on the button instead of typing the MRI